pkg/skaffold/constants/constants.go

```diff
@@ -43,3 +43,17 @@ const (
 	// DefaultKanikoImage is v0.1.0
 	DefaultKanikoImage = "gcr.io/kaniko-project/executor:v0.1.0@sha256:501056bf52f3a96f151ccbeb028715330d5d5aa6647e7572ce6c6c55f91ab374"
 )
+
+var Labels = struct {
+	TagPolicy     string
+	Deployer      string
+	Builder       string
+	DefaultLabels map[string]string
+}{
+	DefaultLabels: map[string]string{
+		"deployed_with": "skaffold",
```

Comment:
nit: hyphen instead of underscore

https://github.com/kubernetes/website/pull/8703/files?short_path=7b9d071#diff-7b9d071558ae546db530549c8c8542a1